base/process.jl

```diff
@@ -647,6 +647,9 @@ Run a command object, constructed with backticks. Throws an error if anything go
 including the process exiting with a non-zero status.
 """
 function run(cmds::AbstractCmd, args...)
+    if isempty(cmds.exec) # Issue 19094
```

Comment:
I would not put the issue number in a comment here – we do that in test code but not in base code.